galaxy.yml

```diff
@@ -8,7 +8,7 @@ namespace: paritytech
 name: chain
```

Comment:
I think this should be put into a `common` or `utils` collection instead of the `chain` collection

Reply:
That's a better way yes. I'll have to modify the GAction as well.

Reply:
Great !